gtests/net/mptcp_unsupported/mptcp_ip_tos_server.pkt

```diff
@@ -0,0 +1,14 @@
+--tolerance_usecs=100000
```

Comment:
@dcaratti : should we have a version in `mptcp/sockopts` showing the current status? I mean: showing that the `[sg]etsockopt()` are showing that the sockopt is not supported so it would fail when it will implemented? (so we don't forget to use this one from the `unsupported` dir?

It is optional, maybe we will not forget about the packetdrill test when implementing this sockopt :)

Reply:
> @dcaratti : should we have a version in `mptcp/sockopts` showing the current status? I mean: showing that the `[sg]etsockopt()` are showing that the sockopt is not supported so it would fail when it will implemented? (so we don't forget to use this one from the `unsupported` dir?

not sure if it's worth doing. I initially thought to test "unsupported" sockopt only for those that are unsupported by design on MPTCP (and, I thought `TCP_INQ` was a good example :-) ).

Reply:
Yes sure, we can only have one and remember about adding new packetdrill tests when supporting more sockopts :)